app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -149,29 +159,51 @@ def handle_bad_request_error(err, enrollment)
     else
       analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_exception(
         **enrollment_analytics_attributes(enrollment, complete: false),
-        **response_analytics_attributes(err.response),
-        reason: 'Request exception',
+        **response_analytics_attributes(response),
         exception_class: err.class.to_s,
         exception_message: err.message,
+        reason: 'Request exception',
+        response_status_code: err.response_status,
       )
       enrollment_outcomes[:enrollments_errored] += 1
```

Comment:
Can we report the 4xx error here to New Relic?

Reply:
Yep. It was being reported up the callstack but I agree it makes sense to report it here. See [17486f2](https://github.com/18F/identity-idp/pull/7212/commits/17486f2e9945e6d0fe2130cc314251183a294126)

Reply:
@sheldon-b I'm not seeing the part you're talking about. This branch of logic only appears to report to CloudWatch, not New Relic.

Reply:
Oops you're right! I mistook where this comment was. Good eye, I'll add that in in a fresh PR